content/http-handler-abi.md

```diff
@@ -156,10 +158,10 @@ This parameter supports the most common case of retrieving a header value by
 name. However, there are some subtle use cases possible, particularly helpful
 for WebAssembly performance:
 
-- re-using a buffer for reading header or path values (`buf`).
-- growing a buffer only when needed (retry with larger `buf_limit`).
-- avoiding copying invalidly large header values (`buf_limit`).
-- determining if a header exists without copying it (`buf_limit=0`).
+* re-using a buffer for reading header or path values (`buf`).
```

Comment:
This was autogenerated by IDE I guess to keep consistency with other lists (e.g. line 90). Can rollback if you prefer so @codefromthecrypt